extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -124,6 +124,9 @@ class AmpLightbox extends AMP.BaseElement {
         'keydown', this.boundCloseOnEscape_);
     this.getViewport().enterLightboxMode();
 
+    if (this.isScrollable_) {
```

Comment:
just add a comment that iOS has a bug where it may not repaint the screen in certain situations if webkitOverflowScrolling is present and the work around is to add/remove webkitOverflowScrolling on open/close of the lightbox.